packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -3902,7 +3906,8 @@ describe('SnapController', () => {
           ) {
             return true;
           }
-          return originalCall(method, ...args);
+
+          return (originalCall as any)(method, ...args);
```

Comment:
Why?

Reply:
`originalCall` doesn't take variable number of arguments, latest TSC doesn't allow variadic arguments into non-variadic functions. I'll add a comment.